intro-to-R-tidyverse/01-introduction-base-R.Rmd

```diff
@@ -0,0 +1,426 @@
+
+## The RStudio Environment
+
+The RStudio environment has four main **panes** (their specific location can be changed under Tools -> Global Options -> Pane Layout).
+![RStudio Appearance](screenshots/rstudio-panes.png) 
+
+1. The **editor** pane is where you can write R scripts and other documents. 
+This is your _text editor_, which will allow you to save your R code for future use. 
+Note that change code here will not run automatically until you runu it. 
+
+2. The **console** pane is where you can _interactively_ run R code. 
+
+3. The **environment** pane primarily displays the variables, sometimes known as _objects_ you define during a given R session. 
+
+4. The **help viewer** pane has several tabs all of which are pretty important:
+    + The **files** tab shows the structure and contents of your _current directory_ (i.e. folder).
+    + The **plots** tab will reveal plots when you make them
+    + The **packages** tab shows which installed packages have been loaded into your R session
+    + The **help** tab will show the help page when you look up a function
+    + The **viewer** pane will reveal compiled RMarkdown documents 
+
+### Comments
+
+R will _ignore_ comments, which are indicated with hashtags. 
+Arguably the __most important__ aspect of your coding is comments: Small pieces of explanatory text you leave in your code to explain what the code is doing and/or leave notes to yourself or others. 
+Comments are invaluable for communicating your code to others, but they are most important for **Future You**. 
+Future You comes into existence about one second after you write code, and has no idea what on earth Past You was thinking. 
+Help out Future You by adding lots of comments! 
+Future You next week thinks Today You is an idiot, and the only way you can convince Future You that Today You is reasonably competent is by adding comments in your code explaining why Today You is actually not so bad.
+
+## Basic Calculations
+
+### Mathematical operators
+
+The most basic use of R is as a regular calculator:
+
+| Operation | Symbol |
+|-----------|--------|
+| Add  | `+` | 
+| Subtract  | `-` | 
+| Multiply  | `*` | 
+| Divide  | `/` | 
+| Exponentiate | `^` or `**` | 
+
+For example, we can do some simple multiplication like this. 
+In this and following code chunks, the section with the gray background is R code, and the following white-background chunk is the code's output.
+
+```{r}
+5 * 6
+```
+
+### Defining and using variables 
+
+To define a variable, we use the _assignment operator_ which looks like an arrow: `<-`, for example `x <- 7` takes the value on the right-hand side of the operator and assigns it to the variable name on the left-hand side. 
+The opposite is also possible, e.g. ` 7 -> x`, as long as the arrow is pointing towards the variable name.
```

Comment:
Okay agreed. This is a thing I've noticed Stephanie does and I don't necessarily like it but I thought maybe it would solidify the arrow concept of assignment. But I agree with you, don't want to encourage this.